pkg/service/nodes.go

```diff
@@ -3,20 +3,24 @@ package service
 import (
 	"context"
 	"fmt"
+	"os"
 	"os/exec"
 	"strings"
 	"time"
 
-	"github.com/avast/retry-go"
-	"github.com/keikoproj/lifecycle-manager/pkg/log"
+	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/api/core/v1"
```

Comment:
aren't these same imports with different aliases?

Reply:
That's right. Missed this while adding the drainNode code. Thanks for catching this.